SPECS/python-tensorboard/python-tensorboard.spec

```diff
@@ -18,7 +18,7 @@ BuildRequires:  python3-setuptools
 BuildRequires:  python3-pip
 BuildRequires:  python3-wheel
 BuildRequires:  python3-six
-BuildRequires:  bazel = 5.3.0
+BuildRequires:  bazel
```

Comment:
Double-checking: no file needed to be removed or config updated here?

Reply:
tensorboard did not have the .bazelversion file by default.